index.html

```diff
@@ -250,7 +247,8 @@ <h3>LET’S CONNECT</h3>
       <!--End of about section-->
        <!--Start of Contact Page-->
        <section class="contact-page">
```

Comment:
- [ ] Please I highly suggest implementing CSS Grid and Flexbox properties in your `contact-page` class which is the contact section which helps improves mobile, tablet and desktop responsiveness for a better user experience. 😄